### DISCUSSION TITLE
IIS Deployment Next Js Application with Next Start

### DISCUSSION THREAD

**Discussion by karahanharunn** [vercel/next.js#34276] [Help]
Hi,

I want to deploy my next js nonstatic app with IIS.

But I have an interesting bug. Firstly I don't know IIS very well.

I am getting build then I see this build folder

![image](https://user-images.githubusercontent.com/32582360/153721339-8e30e118-eab5-444b-93bf-7dd4ba204059.png)

When I open the page on the client-side my .next folder reset. I couldn't understand the problem.

![image](https://user-images.githubusercontent.com/32582360/153721366-7664100f-d3a3-42aa-9ff5-df99952711bc.png)

Why I can't update my build folder? Did anyone face this issue before?

**Comment by AEidSPL** [comment]
Check [this](https://dev.to/petereysermans/hosting-a-node-js-application-on-windows-with-iis-as-reverse-proxy-397b) guide it works with me to run next project with next start
